Dockerfile

```diff
@@ -44,6 +46,9 @@ RUN apt-get update && \
 # Fix to get tcpdump working
 RUN mv /usr/sbin/tcpdump /usr/bin/tcpdump
 
+# Upgrade gRPC python bindings
+RUN pip install --upgrade grpcio
```

Comment:
do you know why this is needed? seems to be you could end up with incompatible server & client versions.

Reply:
Without this, I was unable to use the p4runtime Python library from p4lang/tutorials. If you comment this line and rebuild the image, you will get this error:

```
Traceback (most recent call last):
  File "/p4app/main.py", line 1, in <module>
    from p4app import P4Mininet
  File "/scripts/p4app.py", line 7, in <module>
    from p4runtime_switch import P4RuntimeSwitch
  File "/scripts/p4runtime_switch.py", line 31, in <module>
    import p4runtime_lib.bmv2
  File "/scripts/p4runtime_lib/bmv2.py", line 15, in <module>
    from switch import SwitchConnection
  File "/scripts/p4runtime_lib/switch.py", line 155, in <module>
    class GrpcRequestLogger(grpc.UnaryUnaryClientInterceptor,
AttributeError: 'module' object has no attribute 'UnaryUnaryClientInterceptor'
```

Reply:
Then I would actually recommend updating the `GrpcRequestLogger` code to work with the supported gRPC version.

Reply:
I think we discussed this problem already. This code is from the p4lang/tutorials p4runtime exercise. It seems to depend on a later version of the grpcio python library. Is this the case? Or are you installing a different version of grpcio that removed `UnaryUnaryClientInterceptor`? Do you have any code examples that use a different logger? That would be helpful.

Reply:
We are supporting a much older version of grpc, that probably doesn't have `UnaryUnaryClientInterceptor` yet. You can check if there is an equivalent in grpc 1.3.2. Or you can keep using a more recent version. If the version you are currently using works fine though, you should probably hardcode that version number in the Dockerfile to guarantee that pip keep installing that version specifically in the future.

Reply:
I couldn't find a straight-forward way to get the same functionality as 1.3.2. Good point about fixing the version. I have fixed the version to 1.15.0